test/proxy/utils/UUPSUpgradeable.test.js

```diff
@@ -71,12 +71,6 @@ contract('UUPSUpgradeable', function () {
     );
 
     const receipt = await legacyInstance.upgradeTo(this.implInitial.address);
-
-    const UpgradedEvents = receipt.logs.filter(
-      ({ address, event }) => address === legacyInstance.address && event === 'Upgraded',
-    );
-    expect(UpgradedEvents.length).to.be.equal(1);
```

Comment:
Why are we removing just this instead of the whole test and the UUPSLegacy mock?

Reply:
Is there a reason to keep explicit support for the old version in the first place? I see in the `UUPSUpgradeableLegacyMock` that is for contracts pre-4.5.

I see the use case of just using the upgrade utils, so I'm not strong about removing these tests but I'd suggest being clear and keeping the previous lines but checking for `2` events emitted instead.

Reply:
I just removed it.